jekyll/_cci2/server/v4.6/installation/phase-1-prerequisites.adoc

```diff
@@ -142,7 +142,7 @@ The application uses a large number of resources. Depending on your usage, your
 | Kubernetes Version
 
 | 4.6.x
-| 1.26 - 1.29
+| 1.26 - 1.31
```

Comment:
THis can be added to the latest docs too

Reply:
Done. :-)